tests/PhpcrUtils/CndParserTest.php

```diff
@@ -93,22 +98,29 @@ public function testParseError()
         $this->cndParser->parseString($cnd);
     }
 
-    /**
-     * @expectedException \InvalidArgumentException
-     */
     public function testErrorNoFile()
     {
+        $this->expectException(InvalidArgumentException::class);
+
         $this->cndParser->parseFile('/not/found');
     }
 
     /**
      * Have a comment that is never closed. Starting that at the end of the
      * file turned out to be particularly nasty.
-     *
-     * @expectedException \PHPCR\Util\CND\Exception\ScannerException
      */
     public function testScannerErrorComment()
     {
+        /**
+         * Remove this PHP7 only condition after PHPCR-Utils #168
+         * (https://github.com/phpcr/phpcr-utils/issues/168) is closed
+         */
+        if (PHP_MAJOR_VERSION >= 7) {
+            $this->markTestSkipped();
+        }
```

Comment:
please move this into a separate commit, so that we can easily revert it when we fixed the bug